### DISCUSSION TITLE
Full screen header slideshow

### DISCUSSION THREAD

**Discussion by SirCorono** [templaza/astroid-framework#266] [Q&A] [answered]
Hi
I recently found the Astroid-Framework and started using it. What I try to do is something similar like on this template demo: A slideshow that covers the whole screen on the startpage, even the menu.
https://demo.joomlaplates.de/barber/

If I understood the browser inspector correctly, it seems that there are two sections, one is the header and then there is another one "astroid-element-overlay" that lies somehow behind the header and covers the whole screen.

How can I do that? I was not able to merge or nest sections.

Thanks for your help!

**Comment by Chacapamac** [comment, marked as answer]
I @SirCorono 
Have a look here —> https://efoilsurf.ca
I use SmartSlider Pro —> https://extensions.joomla.org/extension/smart-slider-2/
I recommend you use the same.
Before, the slider was Full Browser view on arrival, but we change that ( It”s only a setting on the slider). You also have a FULL Browser view on arrival setting.

You need to put the two top section position absolute to permit the slider section to pass under

I set the top section (Tools) —> “topbar” to
```
body.frontpage section#topbar {
  z-index: 8;
  position: absolute !important;
  width: 100% !important;
  height: 40px !important;
}
```
and the following menu section —> “header-section” to
```
body.frontpage section#header-section {
  position: absolute !important;
  margin-top: 40px !important;
}
```

This slider have a top margin setting that can be negative, you just have to set the margin and the z-index of the slider elements properly.

**Comment by SirCorono** [comment]
Hi @Chacapamac
Thanks for your answer. That helped. z-index and the negative margin was the solution.
Corono

**Reply by Chacapamac** [reply]
Happy to help!
and Happy New Year to the Astroid Community!
